components/TopRatedList.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import useSWR from "swr";
+import styled from "styled-components";
+import { CardImage } from "./CardImage.styled";
+
+const Container = styled.div`
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(auto-fit, minmax(140px, 1fr));
+`;
+
+const List = styled.li`
+  list-style: none;
+`;
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
```

Comment:
if you use the fetcher twice, it can also make sense to outsource the function to a "service" folder and import it into each of the files you need the fetcher. You could think about that too for the refactoring :)